packages/@o3r/pipeline/schematics/ng-add/templates/github/__dot__github/actions/setup/action.yml

```
@@ -0,0 +1,17 @@
+name: Setup and Install
+description: Sets up Node.js and installs dependencies
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
+      with:
+        node-version: 20
```

Should we setup the cache directly in `setup-node` action ?
```suggestion
    - uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
      with:
        node-version: 20
        cache: <%= packageManager %>
```